tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,66 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function update_worker_label(){
+	local workers
+	local label="$1"
+	workers="$(kubectl get nodes --no-headers | grep '\<worker\>' | awk '{ print $1 }')"
```

Comment:
Not all Kubernetes installations will have `worker` on the worker name.

For example, our SEV/SNP jobs run on baremetal machines which got Kubernetes installed with [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm). It is a single-node installation which worker's name is the host name. @ryansavino could you please confirm that information?

The TDX job installs kubernetes via [k3s](https://k3s.io/). Never used k3s but I suppose the worker's name will likewise be the host name. @fidencio am I right?

 We will need to improve the logic to find the workers. Alternatively assume they will be properly labeled by the system admin (if on baremetal system) or the provisioner code (e.g. if on AKS: https://github.com/kata-containers/kata-containers/blob/main/tests/gha-run-k8s-common.sh#L57)